src/components/SelectionList/SelectableListItem.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useCallback} from 'react';
+import {View} from 'react-native';
+import PressableWithFeedback from '@components/Pressable/PressableWithFeedback';
+import SelectCircle from '@components/SelectCircle';
+import TextWithTooltip from '@components/TextWithTooltip';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import BaseListItem from './BaseListItem';
+import type {InviteMemberListItemProps, ListItem} from './types';
+
+function SelectableListItem<TItem extends ListItem>({
```

Comment:
Can we just reuse `RadioListItem` instead of creating a new component? We we do that for the existing `CategoryPicker` [here](https://github.com/Expensify/App/blob/0928292c499eaca989dad00aefbfa41d94885bdf/src/components/CategoryPicker.tsx#L77-L86)?

Reply:
Hi! `RadioListItem` looks like [this](https://github.com/Expensify/App/pull/46197#discussion_r1693146061) that's why we decided to create a new component to make it look more like this [screen shot](https://github.com/Expensify/App/pull/46197#discussion_r1693279782).